### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by alef-garrido**
- Create an npm project with webpack. 
- Write plain HTML markup with minimum styling

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import './style.css';
+import renderScoreList from './utils/list.js';
+import renderForm from './utils/form.js';
+
```
Comment (carlos-ssh) [inline src/index.js:4]: - Kindly fix your linter in your pull reques.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @alef-garrido , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (alef-garrido) [reply src/index.js:4]: Hello Carlos, 
This linter is not something i 've changed, i'm using the same files provided by microverse. 
Do you have any ideas on why the Lighthouse is failing? Have you s

Comment (henatan99) [reply src/index.js:4]: The Linter check has passed. Nice work! :clap:

**Review by henatan99 — APPROVED** [review]
Hi @alef-garrido , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] OPTIONAL: Make sure your PR summary reflects changes you have made. I don't think writing just the project requirements is the best way. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**